WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -918,32 +917,12 @@ private synchronized void savePostToDb() {
     public void onBackPressed() {
         if (isPhotoPickerShowing()) {
             hidePhotoPicker();
```

Comment:
We need a `return` here - we don't want to call `handleBackPressed()` when the photo picker is showing.